upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -924,7 +924,7 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 
 	if b.cluster.Spec.Networking.Cilium != nil {
 		key := "networking.cilium.io"
-		version := "1.6.6-kops.0"
+		version := "1.7.0-kops.1"
```

Comment:
Maybe use something like this here?
https://github.com/kubernetes/kops/blob/91867ce4b5f945d88a09a1424b48d22a76a4617d/upup/pkg/fi/cloudup/bootstrapchannelbuilder.go#L861-L866